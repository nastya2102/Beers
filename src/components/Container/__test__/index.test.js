import React from 'react'
import { render } from '@testing-library/react'
import Container from '../index'

describe('Container', () => {
  test('should render Container component', () => {
    const { container } = render(<Container><span>123</span></Container>)

    expect(container).toBeTruthy()
  })

  test('should match snapshot Container component', () => {
    const { container } = render(<Container><span>123</span></Container>)

    expect(container).toMatchSnapshot()
  })
})
