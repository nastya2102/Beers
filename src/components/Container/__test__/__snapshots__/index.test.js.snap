// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should match snapshot Container component 1`] = `
<div>
  <div
    class="sc-bdfBwQ dVIEiA"
  >
    <span>
      123
    </span>
  </div>
</div>
`;
